scrapy_google_sheets_exporter/gsheets_exporter.py

```diff
@@ -56,8 +58,22 @@ def _store_in_thread(self, file):
         )
         if self.overwrite:
             self.sheet.clear()
-        rows = [row for row in csv_data]
-        self.sheet.append_rows(rows)
+
+        batch = []
+        payload_size = 0
+        for row in csv_data:
+            row_size = sum(len(cell.encode('utf-8')) for cell in row)
```

Comment:
I like this idea but would it work with non string values like numbers or booleans?

Reply:
if `batch_size` is `0` by default `len(batch) >= self.batch_size`  will always be true on each iteration making 1 call per row, which is not the desired.  I think it should be `payload_size >= self.max_payload_size or self.batch_size and len(batch) >= self.batch_size`

Reply:
@VMRuiz oh you're right, I forgot to add the self.batch_size condition back. I also turned cell to string to account for boolean and integers. thanks for catching it!